README.md

```diff
@@ -3,13 +3,26 @@
 ## Release a new version
 
 You can release a new version by creating a tag in all the assisted-installer repositories.
+
 1. Update the assited-installer.yaml with the relevant git hash.
 2. Build the docker image, locally
-```
+
+```bash
 make local-update
 ```
+
 3. Execute the release image
 
 ```shell script
 docker run -v $(pwd)/assisted-installer.yaml:/assisted-installer.yaml -v $HOME/.netrc:/root/.netrc -it assisted-installer-deployment:local -t <tag>
 ```
+
+## Add new signature
+
+Test a new signature against a specific issue
+
+```bash
+./tools/add_triage_signature.py -up "[user:password]" --verbose -i [issue] -us [signature] --dry-run
```

Comment:
There's already a comprehensive description inside the tool arg_parse itself with more information relevant to whoever wants to run this script, might want to include it there instead, and just refer the user to `./tools/add_triage_signature.py --help` here